src/events/reaction/ReactionAddEventHandler.ts

```diff
@@ -48,6 +50,9 @@ export default class ReactionAddEventHandler implements DiscordEventHandler<'mes
 		} else if ( BotConfig.request.logChannel.includes( messageReaction.message.channel.id ) ) {
 			// Handle reopening a user request
 			return this.requestReopenEventHandler.onEvent( messageReaction, user );
+		} else if ( messageReaction.message.author.id === this.botUserId && messageReaction.emoji.name === BotConfig.embedDeletionEmoji ) {
```

Comment:
@violine1101 there is! Thanks for reviewing!

Reply:
Ah, I see, thanks. I would have expected it inside the actual event, but that's fine too.